### ISSUE TITLE
Cuda 11.0 detected but torch only supports: cpu, 10.1, 10.2, 9.2

### ISSUE THREAD

**Issue by ekianjo** [mlverse/torch#285]
As mentioned, using the latest 450.x drivers for Nvidia on Linux, it seems that Torch does not support Cuda 11.x. It seems that this is the case with PyTorch as well. Is there any way to fix it at this stage? Are there any plans for Cuda 11.x in the future?

**Comment by skeydan** [comment]
When are you getting that error? I have CUDA 11.0, and it runs fine with `libtorch` 1.5 ...

**Comment by dfalbel** [comment]
FWIW you can have multiple versions of cuda installed without conflicts. See for example: https://notesbyair.github.io/blog/cs/2020-05-26-installing-multiple-versions-of-cuda-cudnn/

**Comment by ekianjo** [comment]
> When are you getting that error? I have CUDA 11.0, and it runs fine with `libtorch` 1.5 ...

How can I check my libtorch version?

**Comment by dfalbel** [comment]
LibTorch is certainly v1.5 as it's automatically downloaded when you install torch for the first time.

**Comment by skeydan** [comment]
Oops, now I'm trying to install on a new machine and now I see it:

```
#remotes::install_github ...
> library(torch)
Cuda 11.0 detected but torch only supports: cpu, 10.1, 10.2, 9.2
trying URL 'https://download.pytorch.org/libtorch/cpu/libtorch-cxx11-abi-shared-with-deps-1.5.0%2Bcpu.zip'
```

**Comment by skeydan** [comment]
i'll create a PR for this

**Comment by dfalbel** [comment]
Fixed in #291

**Comment by dfalbel** [comment]
I'll close this as we will only support cuda 11.0 when LibTorch officially supports it.
